### PR TITLE
Add some missing key prefix declarations

### PR DESCRIPTION
**Pull request by justbur**

### CODE REVIEW

File: spacemacs/config.el [hunk 1]
```diff
@@ -15,46 +15,54 @@
...
-                                       ("f" .  "files")
+                                       ("c"  . "compile/comments")
+                                       ("C"  . "capture/colors")
+                                       ("e"  . "errors")
+                                       ("f"  . "files")
                                        ("fe" . "files-emacs/spacemacs")
-                                       ("g" .  "git/versions-control")
-                                       ("h" .  "helm/help/highlight")
+                                       ("g"  . "git/versions-control")
+                                       ("gh" . "smeargle")
```
Comment (syl20bnr) [inline spacemacs/config.el]: these 2 prefixes should be in config.el of git layer.

File: spacemacs/config.el [hunk 2]
```diff
@@ -15,46 +15,54 @@
...
-                                       ("n" .  "narrow/numbers")
-                                       ("p" .  "projects")
-                                       ("p$" .  "projects/shell")
-                                       ("q" .  "quit")
-                                       ("r" .  "registers/rings")
-                                       ("s" .  "search/symbol")
-                                       ("sw" .  "search-web")
-                                       ("t" .  "toggles")
+                                       ("i"  . "insertion")
+                                       ("iS" . "auto-yasnippet")
```
Comment (syl20bnr) [inline spacemacs/config.el]: should be in config.el of auto-completion layer

**Comment by syl20bnr** [comment]
Nice :+1:

**Comment by justbur** [comment]
@syl20bnr fixed the two you pointed out. There's probably more that are attached to packages I don't use.

**Comment by syl20bnr** [comment]
Thank you ! :+1: 
Cherry-picked into `develop` branch, you can safely delete your branch.

**Pull request closed without merging**